scripts/jenkins/bench.sh

```diff
@@ -1,11 +1,34 @@
 #!/usr/bin/env bash
-set -euxo pipefail
+set -exo pipefail
 
+## Buildkite specific configuration
+if [ "$CI" == "true" ] ; then
+	export GO111MODULE=on
+
+	# If HOME is not set then use the current directory
+	# that's normally happening when running in the CI
+	# owned by Elastic.
+	if [ -z "$HOME" ] ; then
+		HOME=$(realpath ~)
+		export HOME
+	fi
+
+	# Make sure gomod can be deleted automatically as part of the CI
+	clean_up () {
+	  ARG=$?
+	  # see https://github.com/golang/go/issues/31481#issuecomment-485008558
+	  chmod u+w -R $HOME/go/pkg
+	  exit $ARG
+	}
+	trap clean_up EXIT
+fi
+
+## Bench specific
+set -u
 source ./scripts/jenkins/setenv.sh
 
 export GOFLAGS='-run=NONE -benchmem -bench=.'
 export OUT_FILE="build/bench.out"
 mkdir -p build
 
 make test | tee ${OUT_FILE}
-go-junit-report < ${OUT_FILE} > build/junit-apm-agent-go-bench.xml
```

Comment:
We don't report junit anymore for the benchmarks. In addition, for some reason the environment is not configured correctly:

```


+ go-junit-report
--
  | ./scripts/jenkins/bench.sh: line 33: go-junit-report: command not found
```

Hence I decided to remove this kind of reporting since it's not consumed